test/src/main/java/org/apache/accumulo/test/fate/MultipleStoresIT.java

```diff
@@ -341,8 +341,8 @@ private void testDeadReservationsCleanup(TestStoreFactory<LatchTestEnv> testStor
 
     // Create the new Fate/start the Fate threads (the work finder and the workers).
     // Don't run another dead reservation cleaner since we already have one running from fate1.
-    FastFate<LatchTestEnv> fate2 = new FastFate<>(testEnv2, store2, false, Object::toString,
-        DefaultConfiguration.getInstance());
+    Fate<LatchTestEnv> fate2 =
```

Comment:
Was this code needlessly using FastFate before?   Its not running the dead reservation cleaner, so it does not need to use FastFate?